include/zephyr/net/ieee802154.h

```diff
@@ -28,9 +28,20 @@ extern "C" {
  * @{
  */
 
-#define IEEE802154_MAX_ADDR_LENGTH	8
+#define IEEE802154_MTU			127
+#define IEEE802154_CRC_LENGTH		2
+#define IEEE802154_DGRAM_MTU		(IEEE802154_MTU - IEEE802154_CRC_LENGTH)
```

Comment:
@nandojve  The reason I touched this at all was that the same name was used in two places (CCxxxx driver and IEEE802154 L2) with different meaning (127 vs. 125) which gave a name conflict now that the constant is exposed in userspace. Thinking about it now, the term MTU was probably plain wrong in both cases (should be aMaxPhyPacketSize - MHR - MFR) - maybe even a bug as in case of the CCxxxx driver this constant finds its way via net_if_dev -> net_if directly into layer 3 protocols. I'll have to check this.

Reply:
Accept your proposition. Fixed.